utils.py

```diff
@@ -62,5 +62,8 @@ def semester_date(semester_str):
 	year = int(semester_str.split('_')[1])
 	semester = semester_str.split('_')[0]
 	month = (1 if semester == 'spring' else FALL_START_MONTH)
-	
 	return datetime(year, month, 1)
+
+def semester_to_num(semester_str):
```

Comment:
I feel like this should be in `semesters.py`, and the other deprecated semseter functions need to be removed from `utils.py`.